app/gui/view/component-browser/component-list-panel/grid/src/content.rs

```diff
@@ -0,0 +1,153 @@
+//! A module with useful structures related to the content of
+//! [grid inside the Component List Panel](crate::View).
+
+use crate::prelude::*;
+
+use ensogl_core::data::color;
+use num_enum::IntoPrimitive;
+use num_enum::TryFromPrimitive;
+
+
+
+// ===========================
+// === Content Identifiers ===
+// ===========================
+
+// === SectionId ===
+
+/// A Component Groups List Section identifier.
+#[derive(Copy, Clone, Debug, Default, Eq, Hash, IntoPrimitive, PartialEq, TryFromPrimitive)]
+#[repr(usize)]
+pub enum SectionId {
+    /// The "Popular Tools" section.
+    #[default]
+    Popular    = 1,
+    /// The "Local Scope" section.
+    LocalScope = 2,
+    /// The "Sub-Modules" section.
+    SubModules = 0,
```

Comment:
Why are we using numbers for these enum variants? The mapping is not obvious to me.